reflexio/cli/run_services.py

```
@@ -294,6 +295,19 @@ def should_start_local_embedding_service() -> bool:
     return os.environ.get("CLAUDE_SMART_USE_LOCAL_EMBEDDING") == "1"
 
 
+def _ensure_nextjs_dependencies(project_dir: Path) -> bool:
+    """Install Next.js project dependencies when node_modules is missing."""
+    if (project_dir / "node_modules").exists():
+        return True
+
+    print(f"Installing {project_dir.name} dependencies...")
+    result = subprocess.run(["npm", "install"], cwd=str(project_dir), check=False)
```

Ruff (0.15.14)

[error] Starting a process with a partial executable path

(S607)